distributed/protocol/tests/test_pandas.py

```diff
@@ -15,6 +12,10 @@
 )
 from distributed.utils import ensure_bytes
 
+pd = pytest.importorskip("pandas")
+np = pytest.importorskip("numpy")
```

Comment:
These checks need to be above the `dask.dataframe` import (or add a new skip for `dask.dataframe`)